schema/crdb/dbinit.sql

```diff
@@ -3436,14 +3436,57 @@ STORING (
   time_deleted
 );
 
+/**
+ * A view of the ipv4 nat change history
```

Comment:
Not blocking but a comment of what this union is doing and why it's here would be nice.

Reply:
I actually found a bug in the original view while writing the comments 😭 
So I'm actually very thankful that you had me go back and write some comments 😂